frontend/src/model/Cocktail.ts

```diff
@@ -1,5 +1,5 @@
 export type Cocktail = {
-    strDrink : string
-    strDrinkThumb : string
-    idDrink: string
+    strDrink : string //I know they are stupid variable names,
+    strDrinkThumb : string // but if I change them,
+    idDrink: string  //the page doesn't load :D
```

Comment:
Change them to valid names. And Edit your dummyData to fit these names.

Since you are using an external api, your backend needs to map the names from the api to your names.
We can have a look at this together when your at that point :)